README.md

```diff
@@ -18,29 +18,55 @@ This will also install some Git pre-commit hooks that check any modified files a
 
 ### Regular development
 
-To run all tests and linters:
+TL;DR: All checks are run with:
 ```sh
 npm t
 ```
+#### Dev Entry point
 
-Several npm scripts are available to only run a subset of these, such as:
+To run the development entry point:
 ```sh
-npm run test-only # only tests without linters
-npm run test:unit # only unit tests
+npm run dev
+```
+You can visit the dev entry point at <http://localhost:3000/>.
+Changes to the source files will be applied automatically,
+so you can start this once and then leave it running in the background.
+
+For instructions to see the form in the context of the special page,
+see the [WikibaseLexeme readme](https://gerrit.wikimedia.org/g/mediawiki/extensions/WikibaseLexeme/+/master/README.md).
+
+#### Linting
+The linters, except for TypeScript type checking, can be run with:
+```sh
+npm run lint
 ```
 
 Many linter errors can be fixed automatically:
 ```sh
 npm run fix
 ```
 
-To run the development entry point:
+Static type checking is executed with
 ```sh
-npm run dev
+npm run check-types
 ```
-You can visit the dev entry point at <http://localhost:3000/>.
-Changes to the source files will be applied automatically,
-so you can start this once and then leave it running in the background.
 
-For instructions to see the form in the context of the special page,
-see the [WikibaseLexeme readme](https://gerrit.wikimedia.org/g/mediawiki/extensions/WikibaseLexeme/+/master/README.md).
+#### Testing
+
+Several npm scripts are available to only run a subset of tests, such as:
+```sh
+npm run test-only # only tests without linters
+npm run test:unit # only unit tests
```

Comment:
Maybe we can remove the comment on this line? It’s pretty redundant now that there’s a better explanation of unit tests below, I guess :shrug:

Reply:
True, it is somewhat self-explanatory